larpandoracontent/LArHelpers/LArMCParticleHelper.h

```diff
@@ -274,9 +299,10 @@ class LArMCParticleHelper
      *  @param  pfoList the input list of Pfos
      *  @param  selectedMCParticleToHitsMap the input mapping from selected reconstructable MCParticles to their hits
      *  @param  pfoToReconstructable2DHitsMap the output mapping from Pfos to their reconstructable 2D hits
+     *  @param  foldToPrimaries whether to fold the particle hierarchy back to primaries
      */
     static void GetPfoToReconstructable2DHitsMap(const pandora::PfoList &pfoList, const MCContributionMap &selectedMCParticleToHitsMap,
-        PfoContributionMap &pfoToReconstructable2DHitsMap);
+	PfoContributionMap &pfoToReconstructable2DHitsMap, bool foldBackHierachy);
```

Comment:
Check indentation (and const bool as discussed in implementation file). Update variable name in doxygen @param above, and in other cases here.